projects/plugins/jetpack/tools/webpack.config.extensions.js

```diff
@@ -26,19 +26,24 @@ const CopyBlockEditorAssetsPlugin = require( './copy-block-editor-assets' );
  */
 const editorSetup = path.join( path.dirname( __dirname ), 'extensions', 'editor' );
 const viewSetup = path.join( path.dirname( __dirname ), 'extensions', 'view' );
+const blockEditorDirectories = [ 'blocks', 'plugins' ];
 
 /**
- * Filters block scripts
+ * Filters block block scripts
```

Comment:
> I haven't properly tested this yet, but looking over it, it seems good. A few pedantic nitpick comments

Thanks, I've updated the testing instructions just in case.